txpipe/sysmaps.py

```diff
@@ -204,15 +204,18 @@ def run(self):
                 self.save_map(group, f"g2_{b}", map_pix, g2[b], config)
                 self.save_map(group, f"var_g1_{b}", map_pix, var_g1[b], config)
                 self.save_map(group, f"var_g2_{b}", map_pix, var_g2[b], config)
+                self.save_map(group, f"lensing_weight_{b}", map_pix, counts_g[b], config)
```

Comment:
So.... I naively would have expected "lensing_weight" to mean inverse variance weight, but if I understand correclty this is more "number of objects per pixel", right?